starter-code/iteration-1/main.js

```diff
@@ -0,0 +1,20 @@
+var taskArr = [];
+function addTask(task) {
+    taskArr.push(task);
+    return task;
+}
+function listAllTasks() {
+    console.log(taskArr);
+}
+function deleteTask(task) {
```

Comment:
You can also convert these to ES6 if you want: `const cowFunction = (myParams, GoHere) => {
//make cow go moo
}`